ios/ItaliaApp/Info.plist

```diff
@@ -92,6 +94,18 @@
 		<string>rubicon-icon-font.ttf</string>
 		<string>SimpleLineIcons.ttf</string>
 		<string>Zocial.ttf</string>
+		<string>RobotoMono-Bold.ttf</string>
```

Comment:
why we need Roboto and do we need all these weights? - cc @matteodesanti @fpersico

Reply:
we should try to use a system font for that, also we probably don't need all versions of Roboto, just one.

Reply:
Roboto mono is the monospace version of Roboto, the default android font. According to the new Italia Design Guidelines, Roboto Mono is also the font to use for prices, amounts, and code, because it offers a better readability for numbers. I'd like to be consistent across different operative systems and contexts, but if adding one more font is a problem we can try to use a *monospaced* system font.

Reply:
adding a font is not a problem per se, we just need to strike the right balance in terms of download size - in this PR we're adding **12 new** font files - as I said above, can we add just one version of RobotoMono? (e.g. RobotoMono-Bold) instead of all 10 versions?

Reply:
@fpersico is this the correct way of adding a new font?

Reply:
@fgiobergia you may need to add the font files [here](https://github.com/teamdigitale/italia-app/blob/master/package.json#L110) - cc @fpersico

Reply:
yes, add the file to rnpm than `react-native link`

Reply:
Yep that's what I've done! So, just so we're clear, should I get rid of the unused fonts or should I leave them there for good measure?